### ISSUE TITLE
vale sync fails with invalid cross-device link

### ISSUE THREAD

**Issue by carno** [jdkato/txtlint#432]
- OS: Linux
- Install method: direct download
- Vale version `2.16.1`

When trying to use `vale sync`, every execution (using sample config https://vale.sh/hub/microsoft/) fails with

```shell
vale sync
Downloading packages [0/1] █                                                                                                           0% | 0s
rename /tmp/Microsoft2055408144/Microsoft /home/(…)/style/Microsoft: invalid cross-device link
```

As I understand (I'm not a go programmer) that is actually expected in `os.Rename` because `/tmp` is a `tmpfs` filesystem in my OS.

```shell
df -h
Filesystem                Size  Used Avail Use% Mounted on
/dev/mapper/VGsys-LVroot  196G  172G   15G  93% /
/dev/mapper/VGsys-LVhome  262G   37G  212G  15% /home
(…)
tmpfs                      16G   12M   16G   1% /tmp
```

I assume I'm not the only one who has `/tmp` as `tmpfs` so it would be great if one could pass another directory for temporary download or if `sync` would support moving downloaded files between filesystems.

**Comment by jdkato** [comment]
This should be fixed now.
